eth2deposit/key_handling/key_derivation/tree.py

```diff
@@ -37,8 +37,14 @@ def _parent_SK_to_lamport_PK(*, parent_SK: int, index: int) -> bytes:
     return compressed_PK
 
 
-def _HKDF_mod_r(*, IKM: bytes) -> int:
-    okm = HKDF(salt=b'BLS-SIG-KEYGEN-SALT-', IKM=IKM, L=48)
+def _HKDF_mod_r(*, IKM: bytes, key_info: bytes=b'') -> int:
```

Comment:
What do you think about adding a test vector of that `key_info != b''`?

Reply:
There currently aren't any tests for this so I tested against py_ecc.